Source/WPEFramework/Controller.cpp

```diff
@@ -110,6 +114,8 @@ namespace Plugin {
     {
         ASSERT(_service == service);
 
+        Connector::Connector::Instance().Revoke(service);
```

Comment:
Use the C function i.s.o. the C++ functionality, so remark for this class declaration.

Reply:
Done